bin/whisper-fill.py

```diff
@@ -118,20 +119,27 @@ def fill_archives(src, dst, startFrom):
         startFrom = fromTime
 
 
-def main(argv):
-        if len(argv) != 2:
-                print("usage: whisper-fill.py src dst");
-                print("       copies data from src in dst, if missing")
-                if len(argv) == 1 and (argv[0].lower() == "--help" or argv[0].lower() == "-h"):
-                        sys.exit(0)
+def main():
+        option_parser = optparse.OptionParser(
```

Comment:
@brutasse was working on py3 support on https://github.com/graphite-project/whisper/tree/feature/py3
But it's untouched for a year now. I think we can proceed with optparse for now, and fix it later if needed.